src/control/lib/ui/id_flags_test.go

```diff
@@ -0,0 +1,69 @@
+//
+// (C) Copyright 2021-2022 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package ui_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/google/uuid"
+
+	"github.com/daos-stack/daos/src/control/common/test"
+	"github.com/daos-stack/daos/src/control/lib/ui"
+)
+
+func TestUI_LabelOrUUIDFlag(t *testing.T) {
```

Comment:
Would be good to also add a case for a bad label

Reply:
That's handled in the first test case. The tests in this package are pretty light -- they're to verify the `UnmarshalFlag()` methods rather than exercising the label/uuid parsing stuff.